2.train-VAE/scripts/nbconverted/3.beta_vae_training_extract_model_components.py

```diff
@@ -92,14 +92,14 @@
 # These optimal parameter values were fetched by running "optimize_hyperparameters.py" and then running "fetch_hyper_params.ipynb" to learn the best hyperparamaters to use in the VAE.
 trained_vae = VAE(
     input_dim=subset_train_df.shape[1],
-    latent_dim=100,
-    batch_size=112,
+    latent_dim=10,
```

Comment:
Can you describe how you arrived at these hyperparameters? We can talk tomorrow in person as well :)

Previously, we opted for a model with higher `latent_dim` and higher `beta` (given there were several with very similar hyperparameter sweep values)

Reply:
I ran optimize hyperparameters to obtain those hyperparameters. However, after discussion, we have decided to move forward with altered parameters (`beta = 3`, `latent_dim = 70`, `learning_rate = 0.0001`)